cmd/operator-sdk/test/local.go

```diff
@@ -73,7 +73,7 @@ func newTestLocalCmd() *cobra.Command {
 		"Additional flags to pass to go test")
 	testCmd.Flags().StringVar(&tlConfig.moleculeTestFlags, "molecule-test-flags", "",
 		"Additional flags to pass to molecule test")
-	testCmd.Flags().StringVar(&tlConfig.namespace, "namespace", "",
+	testCmd.Flags().StringVar(&tlConfig.operatorNamespace, "operator-namespace", "",
```

Comment:
We need an entry in the changelog with. See the other ones which were commands breaking changes.